hotsos/plugin_extensions/openstack/summary.py

```diff
@@ -10,27 +10,31 @@ def __summary_release(self):
         return self.release_name
 
     @idx(1)
+    def __summary_days_to_eol(self):
```

Comment:
to keep ordering consistent across plugins can we keep this below services and dpkg (so basically put days_to_eol at 4)